src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -32,6 +28,11 @@
     <Compile Include="..\..\.stylecop\GlobalSuppressions.cs" Link="GlobalSuppressions.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="DurableTask.AzureStorage" Version="1.2.2" />
+    <PackageReference Include="DurableTask.Core" Version="2.0.0.5" />

```

Comment:
It's not necessary to reference DurableTask.Core explicitly. Referencing DurableTask.AzureStorage is sufficient.

Reply:
Done